src/parseFunctions.js

```diff
@@ -1,4 +1,4 @@
-import { keyTypes } from './constants.js';
+import { keyTypes, CMCDheaders } from './constants.js';
 
 const parseQueryToJson = (queryString) => {
```

Comment:
Let's change the name of this function to `parseQueryparamToJSON`, so that there's consistency in the capital letters of the _JSON_ word along the file. Also, it's _queryparam_.

Reply:
done ✅